### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'mediumTimeFormat')

### ISSUE THREAD

**Issue by poypoyan** [salesforce/lwc#2946]
### Description
I will actually report/suggest a lightning-base-component feature, but upon setting up the demo in webcomponents.dev, I did not expect this error:
When lightning-input is present in html, the error in title shows.

#### Steps to Reproduce
1. Create new LWC in webcomponents.dev.
2. Just add
```html
<lightning-input
    	type="text"
      label="Type Text"
    ></lightning-input>
```
pretty much anywhere in html.
3. Run

#### Expected Results

Should have no error

#### Actual Results

The error in title is thrown

### Browsers Affected

Chrome 103.0.5060.134

**Comment by bashmish** [comment]
Debugged this with @gluck (credit to him for finding the root cause) on our side side (we work on webcomponents.dev).

Turns out it's a bug in latest [lightning-base-components@1.15.3-alpha](https://www.npmjs.com/package/lightning-base-components/v/1.15.3-alpha), the imports from `lightning/i18nCldrOptions` or `lightning/i18nService` are broken, `formattingOptions` can't be imported.

**Comment by Templarian** [comment]
@poypoyan We're looking into the latest release to see if something was missed. Thanks for reporting this.

**Comment by Templarian** [comment]
@poypoyan @bashmish This should be resolved in `1.15.4-alpha` now.
